### ISSUE TITLE
Failure in healthcheck when testing #1154

### ISSUE THREAD

**Issue by lawrencegripper** [opencb/opencga#1158]
One thing to note is that the healthcheck did seem to fail on my instance once deployed to this is likely something that needs to be reviewed. 

```
{"apiVersion":"v1","time":-1,"warning":"","error":"true","queryOptions":{"metadata":true,"skipCount":true,"limit":2000},"response":[{"id":"Status","dbTime":0,"numResults":-1,"numTotalResults":-1,"warningMsg":"","errorMsg":"DEPRECATED: HBase Database connection test failed","resultType":"","result":[{"Solr":"OK","CatalogMongoDB":"OK","VariantStorage":"KO","VariantStorageId":"hadoop"}]}]}
```

This may relate to an issue with my cluster, I'll redeploy and if it still fails I'll track as an issue

```
ERROR OpenCGAWSServer:153 - HealthCheck : {Solr=OK, CatalogMongoDB=OK, VariantStorage=KO, VariantStorageId=hadoop}
2019-02-14 16:18:12 [http-nio-8080-exec-8] INFO  OpenCGAWSServer:593 - ERROR [500], -1ms, num: -1, GET: http://10.0.0.10/opencga/webservices/rest/v1/meta/status, {}, {"metadata":true,"skipCount":true,"limit":2000}
2019-02-14 16:18:13 [http-nio-8080-exec-10] INFO  OpenCGAWSServer:375 - GET: http://10.0.0.10/opencga/webservices/rest/v1/meta/status, {}, {"metadata":true,"skipCount":true,"limit":2000}
2019-02-14 16:18:13 [http-nio-8080-exec-10] INFO  OpenCGAWSServer:102 - HealthCheck results without cache!
2019-02-14 16:18:14 [http-nio-8080-exec-10] INFO  RecoverableZooKeeper:120 - Process identifier=hconnection-0x7c21e59a connecting to ZooKeeper ensemble=localhost:2181
2019-02-14 16:18:14 [http-nio-8080-exec-10] INFO  ZooKeeper:438 - Initiating client connection, connectString=localhost:2181 sessionTimeout=90000 watcher=org.apache.hadoop.hbase.zookeeper.PendingWatcher@1c72ffd3
2019-02-14 16:18:14 [http-nio-8080-exec-10-SendThread(localhost:2181)] INFO  ClientCnxn:975 - Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
2019-02-14 16:18:14 [http-nio-8080-exec-10-SendThread(localhost:2181)] WARN  ClientCnxn:1102 - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2019-02-14 16:18:14 [http-nio-8080-exec-9] INFO  OpenCGAWSServer:375 - GET: http://10.81.152.47:8080/opencga/webservices/rest/v1/meta/status, {}, {"metadata":true,"skipCount":true,"limit":2000}
2019-02-14 16:18:14 [http-nio-8080-exec-10] WARN  RecoverableZooKeeper:272 - Possibly transient ZooKeeper, quorum=localhost:2181, exception=org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
2019-02-14 16:18:15 [http-nio-8080-exec-10-SendThread(localhost:2181)] INFO  ClientCnxn:975 - Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
2019-02-14 16:18:15 [http-nio-8080-exec-10-SendThread(localhost:2181)] WARN  ClientCnxn:1102 - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2019-02-14 16:18:15 [http-nio-8080-exec-10] WARN  RecoverableZooKeeper:272 - Possibly transient ZooKeeper, quorum=localhost:2181, exception=org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
2019-02-14 16:18:15 [http-nio-8080-exec-10] ERROR RecoverableZooKeeper:274 - ZooKeeper exists failed after 1 attempts
2019-02-14 16:18:15 [http-nio-8080-exec-10] WARN  ZKUtil:423 - hconnection-0x7c21e59a0x0, quorum=localhost:2181, baseZNode=/hbase Unable to set watcher on znode (/hbase/hbaseid)
org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:99)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	
```

**Comment by lawrencegripper** [comment]
I'm redeploying now to double check that this isn't due an issue I created while testing on my instance

**Comment by lawrencegripper** [comment]
Yup looks like it was an issue with my docker instances on the VMSS, starting them fixed the issue. 

I'd like to add a `HEALTHCHECK` status to the `Tomcat` image similar to the work in #1121

**Comment by lawrencegripper** [comment]
Resolved by #1160
